docs/reference/schemas/resource/manifest/provider.md

```diff
@@ -1,38 +1,38 @@
 ---
-description: JSON schema reference for the 'provider' property in a DSC Resource manifest
+description: JSON schema reference for the 'adapter' property in a DSC Resource manifest
```

Comment:
This file should be renamed to `adapter.md` and will need to be updated to point at the canonical URI representing the new schema after this breaking change.